### DISCUSSION TITLE
Cook'n Migration

### DISCUSSION THREAD

**Discussion by regularguy01** [mealie-recipes/mealie#2526] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

To Migrate from Cook'n Recipe manager to Meale.  Cookn does provide their format (.dvo), Html file, text and pdf export.  I tried the HTML file and it doesn't work.  it sees the file as file:///somerecipe.html.

I will be glad to provide any cookn type file you might need.
[HTML_files.zip](https://github.com/mealie-recipes/mealie/files/12455917/HTML_files.zip)
Added just example HTML files and its structure.

### Please provide a concise description of the feature that would resolve your issue.

To have a migrator.  Found in the Migrator page.  That imports all cookbooks, recipes and images form Cookn to Mealie.
link to cookn https://www.dvo.com/recipe-organizer.php

### Please consider and list out some caveats or tradeoffs made in your design decision

Not a coder.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by jeffgallini** [comment]
I'm looking for the same feature currently
